spec/models/edition_spec.rb

```diff
@@ -19,15 +19,40 @@
     end
 
     it "allows 'published' state" do
-      edition = Edition.new(state: 'published')
+      edition = Edition.new(
+        state: 'published',
+        approvals: [Approval.new(user:User.first)],
+      )
+      edition.valid?
+
+      expect(edition.errors.full_messages_for(:state).size).to eq 0
+    end
+
+    it "allows 'review_requested' state" do
+      edition = Edition.new(
+        state: 'review_requested',
+        approvals: [Approval.new(user:User.first)],
+      )
+      edition.valid?
+
+      expect(edition.errors.full_messages_for(:state).size).to eq 0
+    end
 
+    it "allows 'approved' state" do
+      edition = Edition.new(
+        state: 'approved',
+        approvals: [Approval.new(user:User.first)],
+      )
       edition.valid?
 
       expect(edition.errors.full_messages_for(:state).size).to eq 0
     end
 
     it "does not allow arbitrary values" do
-      edition = Edition.new(state: 'supercharged')
+      edition = Edition.new(
+        state: 'supercharged',
+        approvals: [Approval.new(user:User.first)],
```

Comment:
Sorry, I'm not following: we're testing errors on the state field, how is that field affected by the presence of approvals?